spec/models/tank_spec.rb

```diff
@@ -4,6 +4,7 @@
   it "Tank has associations" do
     is_expected.to belong_to(:facility).optional
     is_expected.to belong_to(:organization)
+    is_expected.to have_many(:post_settlement_inventories)
```

Comment:
This can safely be removed. `PostSettlementInventory` is one of the old models that was removed in #375